File: C programming/C_Programs/Assigment 11/Q1.c
/*Q1 fund the maximum and minimum nuber in aarry*/

#include<stdio.h>
void main(){
            int arr[5];

            printf("Enter the  number\n");
            for(int i = 0; i<5;i++)
            {
                scanf("%d",&arr[i]);
            }
                
             for(int i = 0; i<5;i++)
            {
            printf("%d",arr[i]);
            }
           
             int max = arr[0];
             int min = arr[0];
             for (int  i = 1; i < 5; i++)
             {
                if(arr[i]> max){
                     max = arr[i];
                }
                if(min > arr[i]){
                    min =arr[i];
                }
             }
             printf("\n max number:%d ",max);
             printf("\n min number:%d ",min);
             

}

File: Core Java/Core_java_Revision/17th sept/Array/Q1.java
//1.Find minimum and maximum number in array.

class MinMax{
public static void main(String[] args){

int [] arr = {10,20,30,4,2,1};
int max = arr[0];
int min = arr[0];
for(int i = 1;  i < arr.length; i++){
    if( arr[i] < min){
        min = arr[i];}
    if(arr[i] > max){
     max=arr[i];}

}//forlopp end
        System.out.println("Minimum = " + min);
        System.out.println("Maximum = " + max);








}}